appscale/tools/agents/gce_agent.py

```diff
@@ -621,40 +622,33 @@ def assert_required_parameters(self, parameters, _):
         present, or if the client_secrets parameter refers to a file that is not
         present on the local filesystem.
     """
-    is_autoscale = parameters['autoscale_agent']
+    is_autoscale_agent = parameters.get('autoscale_agent', False)
 
     # Make sure the user has set each parameter.
     for param in self.REQUIRED_CREDENTIALS:
       if not self.has_parameter(param, parameters):
         raise AgentConfigurationException('The required parameter, {0}, was' \
           ' not specified.'.format(param))
 
-    # For validating instances being created during autoscaling, check that the
-    # oauth2 storage location is valid.
-    if is_autoscale in ['True', True]:
-      if not os.path.exists(self.OAUTH2_STORAGE_LOCATION):
-        raise AgentConfigurationException('Could not find your signed OAuth2' \
-          'file at {0}'.format(self.OAUTH2_STORAGE_LOCATION))
-      return
-
-    # Next, make sure that either the client_secrets file or the oauth2
-    # credentials file exists.
-    credentials_file = parameters.get(self.PARAM_SECRETS) or parameters.get(
-      self.PARAM_STORAGE)
-    if not os.path.exists(os.path.expanduser(credentials_file)):
-      raise AgentConfigurationException('Could not find your credentials ' \
-        'file at {0}'.format(credentials_file))
-
-    # TODO: Remove this warning once service accounts have been fully tested.
-    secrets_location = os.path.expanduser(parameters[self.PARAM_SECRETS])
-    secrets_type = GCEAgent.get_secrets_type(secrets_location)
-    if (secrets_type == CredentialTypes.SERVICE and
-        not parameters[self.PARAM_TEST]):
-      response = raw_input('It looks like you are using service account '
-        'credentials, which are not currently supported for cloud '
-        'autoscaling.\nWould you like to continue? (y/N)')
-      if response.lower() not in ['y', 'yes']:
-        raise AgentConfigurationException('User cancelled starting AppScale.')
+    # Determine if credentials file exists
+    if is_autoscale_agent:
+      # AppScale VM holds creds files in /etc/appscale/ directory
+      has_oauth2_storage = os.path.exists(self.OAUTH2_STORAGE_LOCATION)
+      has_client_secrets = os.path.exists(self.CLIENT_SECRETS_LOCATION)
+      if not has_oauth2_storage and not has_client_secrets:
+        raise AgentConfigurationException(
+          'Can\'t find neither OAuth2 file at {} nor client secrets file at {}'
```

Comment:
Maybe it would be better to change the wording to "Could not find" instead of "Can't"?